src/commands/upload.yml

```diff
@@ -13,4 +13,5 @@ steps:
         AVIATOR_API_TOKEN: <<parameters.aviator_api_token>>
         ASSETS: <<parameters.assets>>
       name: Upload junit xml files
+      when: always
```

Comment:
what does this do?

Reply:
it makes sure that this command is always run, even if a previous step failed (eg, if a test suite fails we still want to upload the xml files)